lib/sensu-plugins-http/common.rb

```diff
@@ -0,0 +1,30 @@
+
+module Common
+  def initialize
+    super()
+    aws_config
+  end
+
+  def aws_config
+    Aws.config.update(
```

Comment:
That's quite a unique usecase. Mind explaining how this works a bit more in detail, and add a section to the README.md.